openshift/release/generate-release.sh

```diff
@@ -8,6 +8,7 @@ GITHUB_ACTIONS=true $(dirname $0)/../../hack/update-codegen.sh
 git apply openshift/patches/disable-ko-publish-rekt.patch
 git apply openshift/patches/override-min-version.patch
 git apply openshift/patches/use_kafkacat_from_ocp.patch
+git apply openshift/patches/preserve_match_labels_webhook.patch
```

Comment:
Where's this file?

Reply:
https://github.com/openshift-knative/eventing-kafka-broker/blob/release-v1.4/openshift/patches/preserve_match_labels_webhook.patch